tools/WrapExecutableLinker.sh

```diff
@@ -7,13 +7,13 @@ temp_files=()
 trap 'rm -r "${temp_files[@]}"' EXIT
 
 pushd @CMAKE_SOURCE_DIR@ >/dev/null
-git_description="@GIT_DESCRIPTION@"
-git_branch="@GIT_BRANCH@"
-if [ -z "$git_description" ]; then
-    git_description="NOT_IN_GIT_REPO"
```

Comment:
This changes the descriptions in a non-git build from "NOT_IN_GIT_REPO" to an empty string.  I don't think that's necessarily a problem, but was the change intentional?

Reply:
Yea I'm aware of the change. I thought "NOT_IN_GIT_REPO" is not necessarily correct because it might be in a git repo but git is not installed for example, so I just kept it at the empty string. No strong preference though..